docker/bin/test_self

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+echo "--- Buildkite package :package:"
+R CMD build /src
+VERSION=$(grep "^Version" /src/DESCRIPTION | sed 's/Version: //')
```

Comment:
Consider also getting PACKAGE out of the package the same way, so no use of hintr, then this is reusable when we need it later

Reply:
Thanks, that's much better